{{cookiecutter.name}}/.golangci.yml

```diff
@@ -35,3 +47,9 @@ linters-settings:
       - github.com/satori/go.uuid
     packages-with-error-message:
       - github.com/satori/go.uuid: "Unsafe package, prefer github.com/gofrs/uuid"
+
+issues:
+  exclude-rules:
+    - path: '(.+)_test\.go'
+      linters:
+        - funlen
```

Comment:
Should we add cyclop, gocyclo here too?

Reply:
cyclop can disable it's self via the `skip-tests` flag which I've set. I'll add gocyclo here too